### ISSUE TITLE
Merge with redis cluster.

### ISSUE THREAD

**Issue by yihuang** [mpalmer/redis#5]
Since redis 3.0 is released, i guess it's about time to merge nds. Don't known do you have any plan on this? @mpalmer

**Comment by mpalmer** [comment]
No, I have no specific plans.  If NDS is deemed suitable for inclusion in core, I'm happy to assist with that work.

**Comment by 72squared** [comment]
I'd be very interested to see redis-nds be evaluated for inclusion into the mainline of redis. I love redis because of the simplicity of the data structures and the api. Being bound by RAM sucks because often there are only a much smaller subset of keys that are hot. The rest are rarely accessed.

**Comment by mpalmer** [comment]
@72squared, you'll want to go talk to antirez about that.  I have no control over whether something does or doesn't get merged into mainline.
